packages/core/src/components/popover/popover.md

```diff
@@ -71,3 +157,244 @@ Modifiers are the tools through which you customize Popper.js's behavior. Popper
+    content itself (the popover is dismissed the moment the user mouses away from the target).
+</div>
+
+@### Backdrop
+
+The `hasBackdrop` prop governs whether a backdrop appears while the popover is open. When `true`:
+
+- __A transparent backdrop will render beneath the popover__. This backdrop
+  covers the entire viewport and prevents interaction with the document until
+  the popover is closed. This is useful for preventing stray clicks or hovers in
+  your app when the user tries to close a popover.
+- __The popover will receive focus when opened__, allowing for better keyboard accessibility.
+
+Clicking the backdrop will:
+
+- _in uncontrolled mode_, close the popover.
+- _in controlled mode_, invoke the `onInteraction` callback with an argument of `false`.
+
+This backdrop behavior is only available for popovers having `interactionKind={PopoverInteractionKind.CLICK}`.
+An error is thrown if used otherwise.
+
+#### Styling the backdrop
+
+By default, the popover backdrop is invisible, but you can easily add your own styles to
+`.pt-popover-backdrop` to customize the appearance of the backdrop (for example, you could give it
+a translucent background color, like the backdrop for the [`Dialog`](#core/components/dialog) component).
+
+The backdrop element has the same opacity-fade transition as the `Dialog` backdrop.
+
+<div class="pt-callout pt-intent-danger pt-icon-error">
+    <h5>Dangerous edge case</h5>
+    Rendering a `<Popover isOpen={true} hasBackdrop={true}>` outside the viewport bounds can easily break
+    your application by covering the UI with an invisible non-interactive backdrop. This edge case
+    must be handled by your application code or simply avoided if possible.
+</div>
+
+@## Style
+
+@### Dark theme
+
+The `Popover` component automatically detects whether its trigger is nested inside a `.pt-dark`
+container and applies the same class to itself. You can also explicitly apply the dark theme to
+the React component by providing the prop `popoverClassName="pt-dark"`.
+
+As a result, any component that you place inside a `Popover` (such as a `Menu`) automatically
+inherits the dark theme styles. Note that [`Tooltip`](#core/components/tooltip) uses `Popover` internally, so it also benefits
+from this behavior.
+
+This behavior can be disabled when the `Popover` is not rendered inline via the `inheritDarkTheme`
+prop.
+
+@### Sizing
+
+Popovers by default have a `max-width` but no `max-height`. To constrain the height of a popover
+and make its content scrollable, set the appropriate CSS rules on `.pt-popover-content`:
+
+```css.scss
+// pass "my-popover" to `popoverClassName` prop.
+.my-popover .pt-popover-content {
+    max-height: $pt-grid-size * 15;
+    overflow-y: auto;
+}
+```
+
+@reactExample PopoverSizingExample
+
+@### Minimal style
+
+You can create a minimal popover with the `pt-minimal` modifier: `popoverClassName="pt-minimal"`.
```

Comment:
not exactly... `minimal={true}` will also affect the arrow but `popoverClassName="pt-minimal"` will only change the transition.

let's document the prop approach only.